<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Appium-Framework" duration-ms="645" started-at="2015-04-15T17:00:15Z" finished-at="2015-04-15T17:00:16Z">
    <groups>
      <group name="sanity">
        <method signature="InvalidLoginTest.verifyInvalidLoginTest()[pri:0, instance:IntegrationTests.tests.payments.InvalidLoginTest@13fee20c]" name="verifyInvalidLoginTest" class="IntegrationTests.tests.payments.InvalidLoginTest"/>
        <method signature="LoginTest.verifyLoginLogoutTest()[pri:0, instance:IntegrationTests.tests.payments.LoginTest@380fb434]" name="verifyLoginLogoutTest" class="IntegrationTests.tests.payments.LoginTest"/>
        <method signature="SavedConnectionsTest.verifySavedConnectionsFeatures()[pri:0, instance:IntegrationTests.tests.payments.SavedConnectionsTest@5f2050f6]" name="verifySavedConnectionsFeatures" class="IntegrationTests.tests.payments.SavedConnectionsTest"/>
      </group> <!-- sanity -->
    </groups>
    <test name="Mobikwik Automation Tests" duration-ms="645" started-at="2015-04-15T17:00:15Z" finished-at="2015-04-15T17:00:16Z">
      <class name="IntegrationTests.tests.payments.LoginTest">
        <test-method status="SKIP" signature="createDriver(java.lang.String)[pri:0, instance:IntegrationTests.tests.payments.LoginTest@380fb434]" name="createDriver" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="SKIP" signature="instantiateHelpers(java.lang.String)[pri:0, instance:IntegrationTests.tests.payments.LoginTest@380fb434]" name="instantiateHelpers" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- instantiateHelpers -->
        <test-method status="SKIP" signature="verifyLoginLogoutTest()[pri:0, instance:IntegrationTests.tests.payments.LoginTest@380fb434]" name="verifyLoginLogoutTest" duration-ms="1" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginLogoutTest -->
        <test-method status="FAIL" signature="teardown()[pri:0, instance:IntegrationTests.tests.payments.LoginTest@380fb434]" name="teardown" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at UITestFramework.CreateSession.teardown(CreateSession.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- IntegrationTests.tests.payments.LoginTest -->
      <class name="IntegrationTests.tests.payments.InvalidLoginTest">
        <test-method status="SKIP" signature="createDriver(java.lang.String)[pri:0, instance:IntegrationTests.tests.payments.InvalidLoginTest@13fee20c]" name="createDriver" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="SKIP" signature="instantiateHelpers(java.lang.String)[pri:0, instance:IntegrationTests.tests.payments.InvalidLoginTest@13fee20c]" name="instantiateHelpers" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- instantiateHelpers -->
        <test-method status="SKIP" signature="verifyInvalidLoginTest()[pri:0, instance:IntegrationTests.tests.payments.InvalidLoginTest@13fee20c]" name="verifyInvalidLoginTest" duration-ms="1" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyInvalidLoginTest -->
        <test-method status="FAIL" signature="teardown()[pri:0, instance:IntegrationTests.tests.payments.InvalidLoginTest@13fee20c]" name="teardown" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at UITestFramework.CreateSession.teardown(CreateSession.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- IntegrationTests.tests.payments.InvalidLoginTest -->
      <class name="IntegrationTests.tests.payments.SavedConnectionsTest">
        <test-method status="FAIL" signature="createDriver(java.lang.String)[pri:0, instance:IntegrationTests.tests.payments.SavedConnectionsTest@5f2050f6]" name="createDriver" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:15Z" finished-at="2015-04-15T22:30:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[android]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.43.1', revision: '5163bce', time: '2014-09-10 16:27:58'
System info: host: 'shanky-Lenovo-B40-70', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.11.0-26-generic', java.version: '1.8.0_40'
Driver info: driver.version: AndroidDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.43.1', revision: '5163bce', time: '2014-09-10 16:27:58'
System info: host: 'shanky-Lenovo-B40-70', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.11.0-26-generic', java.version: '1.8.0_40'
Driver info: driver.version: AndroidDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:593)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:180)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:240)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:126)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:153)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:109)
	at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:39)
	at UITestFramework.CreateSession.androidDriver(CreateSession.java:132)
	at UITestFramework.CreateSession.createDriver(CreateSession.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:4723 [localhost/127.0.0.1] failed: Connection refused
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:140)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:314)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:363)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:219)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:186)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:215)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:572)
	... 34 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:72)
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:117)
	... 46 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDriver -->
        <test-method status="SKIP" signature="instantiateHelpers(java.lang.String)[pri:0, instance:IntegrationTests.tests.payments.SavedConnectionsTest@5f2050f6]" name="instantiateHelpers" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- instantiateHelpers -->
        <test-method status="SKIP" signature="verifySavedConnectionsFeatures()[pri:0, instance:IntegrationTests.tests.payments.SavedConnectionsTest@5f2050f6]" name="verifySavedConnectionsFeatures" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySavedConnectionsFeatures -->
        <test-method status="FAIL" signature="teardown()[pri:0, instance:IntegrationTests.tests.payments.SavedConnectionsTest@5f2050f6]" name="teardown" is-config="true" duration-ms="0" started-at="2015-04-15T22:30:16Z" finished-at="2015-04-15T22:30:16Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at UITestFramework.CreateSession.teardown(CreateSession.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- IntegrationTests.tests.payments.SavedConnectionsTest -->
    </test> <!-- Mobikwik Automation Tests -->
  </suite> <!-- Appium-Framework -->
</testng-results>
